src/Makefile

```diff
@@ -48,6 +48,9 @@ COMMON_TARGET_DIRS = \
 	build/common             \
 	build/common/NativeTypes \
 
+32_ARGS = -define:ARCH_32
+64_ARGS = -define:ARCH_64
```

Comment:
minor: generally we put numbers at the end, like `ARGS_32`

Reply:
agreed